docs/001_develop/01_development-environment/003_genesis-start/index.mdx

```diff
@@ -11,57 +11,59 @@ tags:
 sidebar_position: 3
 ---
 
-Genesis Start is a small program which allows you to preview an application, including during the development cycle, by running it on your local desktop. 
+Genesis Start enables you to run an application on your local desktop. This is useful when you are developing locally.
 
-From the command line, go to your project's `server` directory, run:
+From the command line, go to your project's **server** directory and run:
```

Comment:
i prefer to use `server` so people know its related to code/files/etc...

Reply:
see my note to you on slack for this

Reply:
changed throughout